services/FileUpdate.proto

```diff
@@ -30,7 +30,7 @@ import "Timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
 /*
-Modify the metadata and/or contents of a file. If a field is not set in the transaction body, the corresponding file attribute will be unchanged. This transaction must be signed by all the keys in the key list of the file being updated. If the keys themselves are being update, then the transaction must also be signed by all the new keys.
+Modify the metadata and/or contents of a file. If a field is not set in the transaction body, the corresponding file attribute will be unchanged. This transaction must be signed by all the keys in the top level of a key list of the file being updated. If the keys themselves are being updated, then the transaction must also be signed by all the new keys.
```

Comment:
Suggest to modify "This transaction must be signed by all the keys in the top level of a key list of the file being updated" to "This transaction must be signed by all KeyList keys"